src/utils/restUtils.ts

```diff
@@ -232,7 +231,7 @@ function sjekkStatuskode(response: Response, url: string) {
                 const queryDivider = r.loginUrl.includes("?") ? "&" : "?";
                 window.location.href = r.loginUrl + queryDivider + getRedirectPath() + "%26login_id=" + r.id;
             } else {
-                logWarningMessage(
+                console.error(
```

Comment:
litt usikker på om denne er god nok? Andre måter å kalle på? For logWarningMessage gjør et postkall som kaller sjekkStatuskode, så da ble det loop..